package.json

```diff
@@ -126,6 +126,7 @@
     },
     "setupFilesAfterEnv": [
       "<rootDir>/scripts/test/testSetup.js"
-    ]
+    ],
+    "snapshotSerializers": ["enzyme-to-json/serializer"]
```

Comment:
This is already in `jest.config.js`, I'll make a note to delete this block. Were something around snapshots not working without this?